pom.xml

```diff
@@ -54,7 +54,7 @@
         <diktat-check.version>1.1.0</diktat-check.version>
         <kotlinpoet.version>1.12.0</kotlinpoet.version>
         <detekt.version>1.20.0</detekt.version>
-        <dokka.version>1.6.21</dokka.version>
+        <dokka.version>1.7.0</dokka.version>
```

Comment:
as I can see it has a dedicated repo in Kotlin ecosystem, but should we use the same version as version of Kotlin?

Reply:
I believe no, because IIRC there are no guarantees that Dokka will have the same release cycle. Maybe, they'll align dokka and Kotlin in the future.